chainer/functions/__init__.py

```diff
@@ -235,6 +235,9 @@
 from chainer.functions.math.prod import Prod  # NOQA
 from chainer.functions.math.scale import scale  # NOQA
 from chainer.functions.math.sign import sign  # NOQA
+from chainer.functions.math.sparse_matmul import sparse_coo_matrix  # NOQA
```

Comment:
Should this class be exposed through the `chainer.functions` module?

Reply:
Perhaps it will be used from users though it is not used even from the tests. I remote it from `__init__.py` now and will add it again when it is certainly necessary.